scripts/merge-settings.sh

```
@@ -0,0 +1,88 @@
+seen = set(base_deny)
+for entry in ext_deny:
+    if entry not in seen:
+        base_deny.append(entry)
+        seen.add(entry)
+
+base['permissions']['deny'] = base_deny
+
+print(json.dumps(base, indent=2, ensure_ascii=False))
+" "$BASE_PATH" "$EXTENSION_FILE" 2>&1)
+
+if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)